docs/build-acceleration.md

```diff
@@ -157,6 +174,27 @@ Looking through the build output with the following points in mind:
 
    ⚠️ Note that the bug described in [Discrepancies between FUTDC logging and build summary](up-to-date-check.md#discrepancies-between-futdc-logging-and-build-summary) may cause the number of succeeded projects to be overstated. This requires changes within Visual Studio, and we hope to fix this in a future release.
 
+## Reference Assemblies
+
+A reference assembly is a DLLs that models the public API of a project, without any actual implementation.
+
+During build, reference assembly timestamps are only updated when their project's public API changes. Incremental build systems use file-system timestamps for many of their optimisations. If project changes are internal-only (e.g. method bodies, private members added/removed, documentation changed) then the timestamp is not changed. Knowing the time at which a public API was last changed allows skipping some compilation.
```

Comment:
```suggestion
During build, reference assembly timestamps are only updated when their project's public API changes. Incremental build systems use file system timestamps for many of their optimisations. If project changes are internal-only (e.g. method bodies, private members added/removed, documentation changed) then the timestamp is not changed. Knowing the time at which a public API was last changed allows skipping some compilation.
```
Nit: I don't think this needs to be hyphenated. Or at least it stood out to me as seeming strange. But English grammar rules are always strange (where they even exist). 😄🤷🏻‍♂️